src/main/java/org/pac4j/dropwizard/Pac4jFactory.java

```diff
@@ -135,6 +138,28 @@ public void setCallbackUrlResolver(
         this.callbackUrlResolver = callbackUrlResolver;
     }
 
+    /**
+     * @since 1.1.0
+     */
+    @JsonProperty
+    public Boolean getSessionEnable() {
```

Comment:
Perhaps prefer naming the option `sessionEnabled`, to match other boolean config options in Dropwizard (e.g. `gzipEnabled`, `cookiesEnable`). It's a bit inconsistent across the codebase (namely with `enableCRLDP` and `enableOCSP`), but the `fooEnabled` structure seems as idiomatic as we can get.